src/allmydata/test/test_eliotutil.py

```diff
@@ -78,6 +78,9 @@
 
 
 class EliotLoggedTestTests(AsyncTestCase):
+    """
+    Tests for the automatic log-related provided by ``EliotLoggedRunTest``.
```

Comment:
Probably mean "provided by AsyncTestCase"?

Reply:
Yea, you're right.